src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/NamedPipeBasedRazorProjectInfoDriver.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Pipes;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.ProjectSystem;
+using Microsoft.AspNetCore.Razor.Utilities;
+using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.ProjectSystem;
+
+internal class NamedPipeBasedRazorProjectInfoDriver : AbstractRazorProjectInfoDriver
+{
+    NamedPipeClientStream? _namedPipe;
+    Task? _readTask;
```

Comment:
Where is this used?

Reply:
This just holds on to the reading task instead of using a discard. I was using it during debugging but it can be removed 🤷